src/ConstructiveSolidGeometry/LinePrimitives/Edge.jl

```diff
@@ -29,7 +29,7 @@ end
 function sample(es::Vector{Edge{T}}; n = 2)::Vector{CartesianPoint{T}} where {T}
     pts = Vector{CartesianPoint{T}}(undef, length(es) * n)
     for i in eachindex(es)
-        pts[i:i+n-1] = sample(es[i], n = n)
+        pts[(2i)-1:(2i)] = sample(es[i], n = n)
```

Comment:
Shouldn't this somehow depend on `n` (`pts[1 + n*(i-1):n*i]`)?

Reply:
Correct, I'll add this.

Reply:
I forced pushed.